iso15118/shared/examples/15118_20_evse_service_config.json

```diff
@@ -177,6 +177,39 @@
           ]
         }
       ]
+    },
+    "6": {
+      "ParameterSet": [
+        {
+          "ParameterSetID": 1,
+          "Parameter": [
+            {
+              "Name": "Connector",
+              "intValue": 2
+            },
+            {
+              "Name": "ControlMode",
+              "intValue": 1
+            },
+            {
+              "Name": "MobilityNeedsMode",
+              "intValue": 1
+            },
+            {
+              "Name": "Pricing",
+              "intValue": 0
+            },
+            {
+              "Name": "BPTChannel",
+              "intValue": 1
+            },
+            {
+              "Name": "GeneratorMode",
+              "intValue": 1
+            }
+          ]
+        }
+      ]
     }
   }
-}
```

Comment:
We could perhaps remove this as -20 DC is not fully supported.

Reply:
Same comment as above - If this is left it in, then the missing parameter set for control mode 2 in DC BPT could be added back in ...